### PR TITLE
Clarify build and install instructions in CONTRIBUTING.org

### PR DESCRIPTION
**Pull request by sean-hut**
Objective:
Help contributors with the preparation of their packages.

Changes:
- Split the instructions on build and install into two subsections.
- Explicitly state directories.

### CODE REVIEW

**Review by alphapapa — CHANGES_REQUESTED** [review]
Here are a few grammatical fixes.

File: CONTRIBUTING.org [hunk 1]
```diff
@@ -200,6 +204,12 @@ version is detected by running ~STABLE=t make recipes/<NAME>~.  The
 version detection can be adjusted by specifying ~:version-regexp~ in
 the recipe (see [[file:README.md#recipe-format][recipe format]] in the README).
 
+**** Test your recipe installs
+
+In your copy of the MELPA repository.  Run the installation check
```
Comment (alphapapa) [inline CONTRIBUTING.org:209]: That's not a complete sentence, and the second is a run-on, but this paragraph seems unnecessary, anyway.  Instead, let's add a paragraph after the following one that says something like:
```
To test a stable release:

1.  Build the recipe by running ~STABLE=t make recipes/<NAME>~.
2.  Install the package from the ~packages-stable~ directory.
```

**Review by alphapapa — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.org [hunk 2]
```diff
@@ -186,10 +186,14 @@ specifying only files relevant to the package.
 
 *** Test your recipe
 
-Please test that the package builds properly by following the steps
-below.
+Please test that the package builds and installs properly by following
+the steps below.
 
-Let ~<NAME>~ denote the filename of the new recipe. Build the recipe
+**** Test your recipe builds
```
[the remark this answers is not in the document]
Comment (sean-hut) [reply CONTRIBUTING.org]: Incorporated into 3b7c979.

File: CONTRIBUTING.org [hunk 3]
```diff
@@ -186,10 +186,14 @@ specifying only files relevant to the package.
...
+
+Navigate to the root directory of your copy of the MELPA repository.
```
[the remark this answers is not in the document]
Comment (sean-hut) [reply CONTRIBUTING.org]: Incorporated into 3b7c979.

File: CONTRIBUTING.org [hunk 4]
```diff
@@ -186,10 +186,14 @@ specifying only files relevant to the package.
...
+
+Let ~<NAME>~ denote the filename of the new recipe.  Build the recipe
```
[the remark this answers is not in the document]
Comment (sean-hut) [reply CONTRIBUTING.org]: Incorporated into 3b7c979.

File: CONTRIBUTING.org [hunk 5]
```diff
@@ -200,6 +204,12 @@ version is detected by running ~STABLE=t make recipes/<NAME>~.  The
 version detection can be adjusted by specifying ~:version-regexp~ in
 the recipe (see [[file:README.md#recipe-format][recipe format]] in the README).
 
+**** Test your recipe installs
```
[the remark this answers is not in the document]
Comment (sean-hut) [reply CONTRIBUTING.org]: Incorporated into 3b7c979.

File: CONTRIBUTING.org (hunk 1 above)
Comment (sean-hut) [reply CONTRIBUTING.org]: Incorporated into 3b7c979.

**Comment by sean-hut** [comment]
Commit 3b7c979 incorporates @alphapapa's feedback and does more clarification and reorganization.

**Comment by alphapapa** [comment]
Unfortunately, that commit has grammatical errors similar to the ones I already offered corrections for.  As well, it does not seem to be better organized than what already exists.  

Perhaps you're not a native English speaker (I'm not sure about this, but given the clues in your writing and your @yandex email address...), in which case I can't fault you for them.  But rather than going back and forth proofreading, it might be better for a native speaker to write it in the first place.  It doesn't seem to make sense for someone like me to proofread it, offer corrections, and then have the corrections rewritten with more mistakes, requiring more proofreading and corrections...it's like an infinite loop.

As well, we should be careful that our changes do not merely reflect our personal preferences.  This documentation is intended to be widely read and it should be easily understood.  If everyone who submitted a new MELPA package came along and rewrote parts of this file to suit their preferences, it would be a lot of churn for no gain.  It may be better to leave it as-is for now.

**Comment by tarsius** [comment]
IMO this makes it harder to read the instructions by merely adding some formatting and repetition. Adding some formatting and even repetition might not be a bad idea in principal, but I don't think that it worked out well here.

**Pull request closed without merging**